app/MMB/src/scenes/appleWallet/DateAndTime.js

```diff
@@ -0,0 +1,57 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { Translation, DateFormatter } from '@kiwicom/mobile-localization';
+import { StyleSheet } from '@kiwicom/mobile-shared';
+
+import WalletText from './WalletText';
+import WalletContext from '../../context/WalletContext';
+
+export default function DateAndTime() {
+  return (
+    <WalletContext.Consumer>
+      {({ selectedSegment }) => {
+        const walletData = selectedSegment || {};
+        const date = new Date(walletData.flightDate || 0);
```

Comment:
Maybe you should not render the date if it is missing, instead of printing 1970.